src/main/java/com/vendor/caterer/interfaces/CatererMapper.java

```diff
@@ -11,8 +12,11 @@
 @Mapper(componentModel = "spring", nullValuePropertyMappingStrategy = NullValuePropertyMappingStrategy.IGNORE)
 public interface CatererMapper {
 
-    public static final CatererMapper mapper = Mappers.getMapper(CatererMapper.class);
+    CatererMapper mapper = Mappers.getMapper(CatererMapper.class);
     Caterer mapCreateRequestToModel(CatererCreateRequest createRequest);
     Caterer mapUpdateRequestToModel(CatererUpdateRequest dto, @MappingTarget Caterer entity);
-
+    Category mapCreateRequestToModel(CategoryCreateRequest createRequest);
```

Comment:
Move category and package to different mapper

Reply:
DONE